src/main/java/com/slimebot/games/Game.java

```diff
@@ -0,0 +1,82 @@
+package com.slimebot.games;
+
+import com.slimebot.main.Main;
+import net.dv8tion.jda.api.entities.channel.middleman.MessageChannel;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+public abstract class Game extends ListenerAdapter {
+
+    public final UUID uuid;
+    public final long channelId;
+    public final long guildId;
+    public long gameMaster;
+    public List<Long> player;
```

Comment:
Probably change this to List<UserSnowflake>. This way you can use getAsMention and don't have to manually build mentions